src/main/webapp/consentGroup/NewConsentGroup.js

```diff
@@ -58,6 +59,8 @@ class NewConsentGroup extends Component {
     this.downloadFillablePDF = this.downloadFillablePDF.bind(this);
     this.submitNewConsentGroup = this.submitNewConsentGroup.bind(this);
     this.uploadFiles = this.uploadFiles.bind(this);
+    this.changeStateSubmitButton = this.changeStateSubmitButton.bind(this);
+    this.toggleSubmitError = this.toggleSubmitError.bind(this);
```

Comment:
Not for this PR - but I do have a question. I've found that it is necessary to bind the functions this way but I do find it verbose and unintuitive boilerplate. I'd love to be able to get rid of this if there's a better way to do this.

Reply:
I'll check if there's a better way to do this binding.